docs/middleware/ics29-fee/fee-distribution.md

```diff
@@ -0,0 +1,90 @@
+<!--
+order: 4
+-->
+
+# Fee Distribution
+
+Learn about payee registration for distribution of packet fees. The following document is intended for relayer operators {synopsis}
+
+## Pre-requisite Readings
+
+* [Fee Middleware](overview.md) {prereq}
+
+Packet fees are divided into 3 distinct amounts in order to compensate relayer operators for packet relaying on fee enabled IBC channels.
+
+- `RecvFee`: The sum of all packet receive fees distributed to a payee for successful execution of `MsgRecvPacket`.
```

Comment:
Seeing as we're saying the `sum of all packet receive fees` it might be worth noting above that a packet can be `topped` up multiple times. wdyt?

Reply:
Maybe that could be added to the section [Escrowing and paying out fees](https://github.com/cosmos/ibc-go/blob/21ac60f50040ffbef6b34e7a8ae3061d4749b224/docs/middleware/ics29-fee/msgs.md#escrowing-and-paying-out-fees) of @charleenfei's PR.

Reply:
Yeah, possibly adding to @charleenfei's PR would be a nice idea and segway into linking to this docs page.

https://github.com/cosmos/ibc-go/pull/1572#discussion_r905222245